src/backer/crawling.py

```diff
@@ -57,8 +57,9 @@ def crawlBlockDo(self, tymth=None, tock=0.0):
                                 trans["block_height"] = block.height
                                 self.ledger.updateTrans(trans)
                 else:
-                    # TODO: Handle for backward
-                    logger.debug("Backward direction")
+                    # Rollback transactions
+                    if self.on_tip and isinstance(block, ogmios.Block):
```

Comment:
We shouldn't have the `on_tip` check here. Rollbacks can occur slightly behind tip